iogt/templates/base.html

```diff
@@ -85,26 +85,34 @@
 
       {% if languages %}
       <div class="languages">
+       {% if languages|length <= 1 %}
```

Comment:
you may be able to write this `{% if languages|length_is:"1" %}

See https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#length-is